crowbar_framework/config/initializers/checks.rb

```diff
@@ -33,8 +33,10 @@
     raise "No local interface is configured with an correct ip address."
   end
 
-  unless network_checks.network_valid?
-    raise "Failed to validate network.json configuration. Please check and fix with yast2 crowbar."
+  if Rails.root.join(".crowbar-installed-ok").exist?
+    unless network_checks.network_valid?
+      raise "Failed to validate network.json configuration. Please check with yast2 crowbar."
+    end
```

Comment:
Actually, that's not correct: if crowbar is installed, then /etc/crowbar/network.json is not used anymore, so the check wouldn't be relevant.